README.md

```diff
@@ -1,8 +1,53 @@
-Telemetry Services
-------------------
+# Mozilla Pipeline Schemas
 
-This repository contains schemas for Mozilla Firefox Telemetry pings.
+This repository contains schemas for Mozilla's data ingestion pipeline and data
+lake outputs.
 
-These schemas are used to validate incoming submissions at ingestion time. The [RapidJSON](http://rapidjson.org/md_doc_schema.html) library is used for JSON Schema Validation. This has implications for what kinds of string patterns are supported, see the `Conformance` section in the linked document for further details.
+The JSON schemas are used to validate incoming submissions at ingestion time.
+The [RapidJSON](http://rapidjson.org/md_doc_schema.html) library is used for
+JSON Schema Validation. This has implications for what kinds of string patterns
+are supported, see the `Conformance` section in the linked document for further
+details.
 
-To learn more about writing JSON Schemas, [Understanding JSON Schema](https://spacetelescope.github.io/understanding-json-schema/index.html) is a great resource.
+To learn more about writing JSON Schemas,
+[Understanding JSON Schema](https://spacetelescope.github.io/understanding-json-schema/index.html)
+is a great resource.
+
+The [Parquet-MR](https://github.com/apache/parquet-format/blob/master/LogicalTypes.md)
+schemas are used for direct to parquet output.
+
+To learn more about Parquet-MR Schemas.
+[Parquet Schema Examples](https://mozilla-services.github.io/lua_sandbox_extensions/parquet/io_modules/lpeg/parquet.html)
+is a limited resource as the full specification is only documented in code.
+
+## Build
+
```

Comment:
I will reword this when mreid provides the artifact clarification.

The [Parquet-MR](https://github.com/apache/parquet-format/blob/master/LogicalTypes.md) schemas are used for direct to parquet output; some examples of Parquet-MR schemas can be found here: [Parquet Schema Examples](https://mozilla-services.github.io/lua_sandbox_extensions/parquet/io_modules/lpeg/parquet.html)